js/map.js

```diff
@@ -17,8 +17,8 @@ function extractLocationFromUrl() {
     var split = hostname.split(".");
     var location = split[0];
 
-    if ((split.length < 3) || (location == "www")) {
-        return "toulouse";
+    if ((split.length < 3) || (location === "www")) {
+        return null;
```

Comment:
I usually don't like returning null value.

When the url don't seems to contains a location, I would expect this method to not return anything  but throw an exception.